tests/components/ct-react-vite/package.json

```diff
@@ -9,19 +9,19 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^6.4.2"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.6.1"
   },
   "devDependencies": {
-    "@types/react": "^17.0.33",
-    "@types/react-dom": "^17.0.10",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.10",
     "@vitejs/plugin-react": "^3.0.0",
     "typescript": "^4.5.4",
     "vite": "^4.0.3"
   },
   "@standaloneDevDependencies": {
-    "@playwright/experimental-ct-react": "^1.22.2",
+    "@playwright/experimental-ct-react": "^2.0.0",
```

Comment:
I don't think we have 2.0 yet, let's keep the versions in sync. It is going to be breaking change, but we are an experimental package.